uiautomator-espresso/python/uiautomator.py

```diff
@@ -0,0 +1,42 @@
+import requests
+import json
+import base64
+
+username = ''
+apiKey = ''
+
+encodeAuth = 'Basic ' + (base64.b64encode((username + ':' + apiKey).encode('utf-8'))).decode('utf-8')
```

Comment:
Why do we use encode and decode at the same time?

Reply:
Python will throw "b 'bmdhbnRsbmd1eWVudGVzdDo0MjMxNThjZS05OTMzLTRkZDMtYjJjZS1kMjhjMWNiNzY2MTk='" (binary) if we do not use decode, and the request header will be incorrect.